### PR TITLE
feat: full background image on onboarding page

### PR DESCRIPTION
**Pull request by omBratteng**
## AS-353 Changes

Extends `onboarding_visual` to accept full background images

<table>
<tr>
 <td>Mobile
 <td>Desktop
<tr>
 <td>
 <td>
</table>

## Testing

This can be tested by setting the `ONBOARDING_VISUAL` value in Growthbook devtools to 
```json
{
  "fullBackground": {
    "mobile": "https://daily-now-res.cloudinary.com/image/upload/s--EwsBTBt6--/f_auto/v1716969841/dailydev_where_developers_suffer_together_mobile_shkn1w",
    "desktop": "https://daily-now-res.cloudinary.com/image/upload/s--r2ffZPB4--/f_auto/v1716969841/dailydev_where_developers_suffer_together_sfvfog"
  }
}
```

### Preview domain
https://as-353-onboarding-background.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/pages/onboarding.tsx [hunk 1]
```diff
@@ -73,6 +73,17 @@ import { LoggedUser } from '@dailydotdev/shared/src/lib/user';
...
+type OnboardingVisual = {
+  showCompanies?: boolean;
+  fullBackground?:
+    | {
+        mobile?: string;
+        desktop?: string;
+      }
+    | false;
+  image?: string;
+};
```
Comment (nensidosari) [inline packages/webapp/pages/onboarding.tsx:85]: Totally fine like this as well, just another suggestion that makes more sense in my opinion, but no strong opinion. Only change if it makes sense for u as well.
```suggestion
type OnboardingVisual = {
  showCompanies?: boolean;
  fullBackground?: boolean;
  image?: string;
  imageMobile?:string
};
```

File: packages/shared/src/lib/featureManagement.ts [hunk 2]
```diff
@@ -24,6 +24,10 @@ const feature = {
   bookmarkOnCard: new Feature('bookmark_on_card', false),
   onboardingVisual: new Feature('onboarding_visual', {
     showCompanies: true,
+    fullBackground: {
+      mobile: cloudinary.onboarding.fullBackground.mobile,
+      desktop: cloudinary.onboarding.fullBackground.desktop,
+    },
```
Comment (nensidosari) [inline packages/shared/src/lib/featureManagement.ts:30]: Here we're setting it as a default value, so let's better leave the last value at the moment if this is going to be an experiment

**Review by nensidosari — APPROVED** [review]
Looks good, one suggestion that for me looks better, but up to u.

Also, from this issue Ante raised here, I believe it's best to check with product for a light mode image?

File: packages/shared/src/lib/featureManagement.ts (hunk 2 above)
Comment (omBratteng) [reply packages/shared/src/lib/featureManagement.ts:30]: As long as they don't get sent from growthbook, it'll be undefined. But I'll set `fullBackground` to false

File: packages/webapp/pages/onboarding.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/webapp/pages/onboarding.tsx:85]: I think it's fine to keep it in `fullBackground`. It defaults to false (or undefined if not specified)

**Pull request merged**